src/pseudopeople/interface.py

```diff
@@ -108,19 +105,19 @@ def _generate_dataset(
 
 def validate_source_compatibility(source: Path):
     # TODO [MIC-4546]: Clean this up w/ metadata and update test_interface.py tests to be generic
-    metadata = source / "metadata.yaml"
-    if metadata.exists():
-        version = _get_data_version(metadata)
-        if version > parse(DATA_VERSION):
+    changelog = source / "CHANGELOG.rst"
```

Comment:
This is reverting a change that was made previously and changing it back to how it was originally.